_pages/about-us/teams/oa.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Office of Acquisition
+---
+
+Welcome to TTS Office of Acquisition (OA) team page!
+
+## Our Mission
+
+We seek to design and evangelize acquisition approaches that are better aligned with current industry best practices to ensure 
+the successful delivery of technology products and services that truly meet the needs of TTS, our agency partners, and the 
+people they serve. 
+
+##Our Approach
+
+In pursuit of our mission, we incorporate the following practices:
+
+* Applying agile and human-centered practices to help government buy better technology
+* Reducing time and costs
+* Starting small and iterating
+* Creating cross-functional teams
+* Focusing on the user need
+* Working in the open and being transparent
+* Following the appropriate federal and agency-specific rules and regulations
+
+## What We Do
+
+We buy digital products and services for TTS as well as for partner agencies (coming soon through assisted acquisitions).
+
+### TTS Internal Procurement
+
+If you are within TTS and your procurement need costs less than the micro-purchase threshold ($10,000), please visit the 
+[Purchase Requests](https://handbook.18f.gov/purchase-requests/) section in the handbook. For SaaS purchases under $10,000 please visit the 
+[Software](https://handbook.18f.gov/software/) section. 
+
+If you are within TTS and your procurement need costs more than the micro-purchase threshold ($10,000), please visit the 
+Purchases over $10,000 section in the handbook.
```

Comment:
This page exists, too, doesn't it?

Reply:
I don't think it's up yet, we are working on the content.